spyderlib/widgets/sourcecode/base.py

```diff
@@ -492,7 +492,8 @@ def copy(self):
         Reimplement Qt method
         Copy text to clipboard with correct EOL chars
         """
-        QApplication.clipboard().setText(self.get_selected_text())
+        if len(self.get_selected_text()) > 0:
+            QApplication.clipboard().setText(self.get_selected_text())

```

Comment:
Thanks for the contribution

In python saying 

`if len(self.get_selected_text()) > 0` is the same as `  if len(self.get_selected_text()):`

So we use the second version (as it is common practice in Python)